### ISSUE TITLE
Regular/big fish miscategorizations

### ISSUE THREAD

**Issue by Ruzihm** [Ottermandias/GatherBuddy#102]
Little Perykos incorrectly listed as non big fish. It counts for the big fish achievements and has a green background.

There are a bunch of regular fish (many of them things like shells) which are listed as big fish. 

It looks like FishData.Unknown5 is used to determine if the fish is big or not but it might be worth looking again for ways to distinguish (maybe unknown3?) 

I'm almost certain this is a known bug but I didn't see an issue anywhere for it. It's entirely possible it can't be addressed due to limitations in Lumina.

In the meantime, I think at least having a place for people to list exceptions could be helpful :)

**Comment by Ottermandias** [comment]
Yeah, this is known, but I don't think there's anything more appropriate in the game files beyond just listing it with the fishdata itself.
At least I haven't found any flag for big fish that comes even close to Unk5 anywhere.

**Comment by Ottermandias** [comment]
I pushed an override. If you notice any other wrongly assigned Big Fish, let me know or fix them up yourself in FishData. :)
